tsconfig.json

```diff
@@ -4,8 +4,8 @@
     "esModuleInterop": true,
     "importHelpers": true,
     "lib": ["esnext"],
-    "module": "commonjs",
```

Comment:
Why is this change needed?

Reply:
Because it was conflicting with esnext. But I've spotted the problem and, like at api.serlo.org, I'm going to upload and use a tsconfig.prod.json.

Reply:
Ah, yeah, that makes sense. Yes, please use the same build toolchain as in api.serlo.org since that worked out well for us in other projects, too :)